docs/validations/validations-rules-reference.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: 'Rules Reference'
+---
+
+# Rules Reference
+
+Rules can be asserted against a gameplay recording to ensure that the recording meets specific criteria. We provide a set of rule types that can be used to construct assertions based on various performance metrics.
+
+Rules belong to a Validation Suite, and are asserted each time the Validation Suite is run.
+
+## Rule Types
+
+### Frames Per Second (with percentile)
+
+Assert that the frames per second of the gameplay recording are above a threshold, within a specified percentile.
+
+### Average Frames Per Second
+
+Assert that the average frames per second of the gameplay recording is above a threshold.
+
+### Maximum Frame Time
+
+Assert that the maximum frame time of the gameplay recording is below a threshold.
```

Comment:
Readers will likely be confused about what frame time is. We should clarify that this represents the time spent calculating and rendering a frame and is the inverse of FPS.

Reply:
good idea added